services/121-service/src/datasource-manage-migrations.ts

```diff
@@ -0,0 +1,12 @@
+import { ORMConfig } from '@121-service/src/ormconfig';
```

Comment:
Add comment to mention that this is only used when running the migrations script from package.json